gnat2goto/driver/tree_walk.adb

```diff
@@ -4282,6 +4371,29 @@ package body Tree_Walk is
          Process_Statement (HSS, Reps);
       end if;
 
+      if Present (Exception_Handlers (HSS)) then
+         declare
+            A_Handler : Node_Id := First (Exception_Handlers (HSS));
```

Comment:
What is this `Exception_Handlers`? I couldn't find anything relevant. Is this supplied from the Gnat frontend somehow?

Reply:
Yeah, this appears to be a gnat-node getter.